# generated by patch-package 6.5.0 on 2021-11-20 09:53:03
#
# command:
#   npx patch-package ts-results --exclude nothing
#
# declared package:
#   ts-results: 3.3.0
#
diff --git a/node_modules/ts-results/esm/index.js b/node_modules/ts-results/esm/index.js
index 5b8900d..0cf894b 100644
--- a/node_modules/ts-results/esm/index.js
+++ b/node_modules/ts-results/esm/index.js
@@ -1,3 +1,3 @@
-export * from './result';
-export * from './option';
+export * from './result.js';
+export * from './option.js';
 //# sourceMappingURL=index.js.map
\ No newline at end of file
diff --git a/node_modules/ts-results/esm/option.js b/node_modules/ts-results/esm/option.js
index c1cb8ec..537163c 100644
--- a/node_modules/ts-results/esm/option.js
+++ b/node_modules/ts-results/esm/option.js
@@ -1,5 +1,5 @@
-import { toString } from './utils';
-import { Ok, Err } from './result';
+import { toString } from './utils.js';
+import { Ok, Err } from './result.js';
 /**
  * Contains the None value
  */
diff --git a/node_modules/ts-results/esm/result.js b/node_modules/ts-results/esm/result.js
index 5a79d2a..0e92b8a 100644
--- a/node_modules/ts-results/esm/result.js
+++ b/node_modules/ts-results/esm/result.js
@@ -1,5 +1,5 @@
-import { toString } from './utils';
-import { None, Some } from './option';
+import { toString } from './utils.js';
+import { None, Some } from './option.js';
 /**
  * Contains the error value
  */
diff --git a/node_modules/ts-results/package.json b/node_modules/ts-results/package.json
index 6f35cbe..889423c 100644
--- a/node_modules/ts-results/package.json
+++ b/node_modules/ts-results/package.json
@@ -2,7 +2,8 @@
   "name": "ts-results",
   "version": "3.3.0",
   "description": "A typescript implementation of Rust's Result and Option objects.",
-  "main": "index.js",
+  "type": "module",
+  "main": "./esm/index.js",
   "module": "./esm/index.js",
   "types": "index.d.ts",
   "keywords": [
